### PR TITLE
Get the tests to pass in travis/netcore/linux

### PR DESCRIPTION
**Pull request by AnthonySteele**
~Work in progress, do not merge yet~

Quick, merge it while it's green

### CODE REVIEW

File: src/NLog.StructuredLogging.Json.Tests/Env.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace NLog.StructuredLogging.Json.Tests
+{
+    public static class Env
+    {
+#if NET462
+        public static bool HasCallSite { get  => true; }
+#else
+        public static bool HasCallSite { get => false; }
```
Comment (martincostello) [inline src/NLog.StructuredLogging.Json.Tests/Env.cs]: Shouldn't this be ```public static bool HasCallSite => true/false;```?

File: src/NLog.StructuredLogging.Json.Tests/Helpers/MapperStandardTests.cs [hunk 2]
```diff
@@ -58,7 +58,10 @@ public void WhenConverted_ThePropertyKeyIsConvertedToProperties()
         [Test]
         public void WhenConverted_TheExceptionIsExpanded()
         {
-            Assert.AreEqual("System.Exception: Outer Exception ---> System.Exception: Inner Exception\r\n   --- End of inner exception stack trace ---", _result["Exception"]);
+            const string expected = "System.Exception: Outer Exception ---> System.Exception: Inner Exception{br}   --- End of inner exception stack trace ---";
```
Comment (martincostello) [inline src/NLog.StructuredLogging.Json.Tests/Helpers/MapperStandardTests.cs]: What about a non-constant string with ```Environment.NewLine``` interpolated into it?

```csharp
Assert.AreEqual($"System.Exception: Outer Exception ---> System.Exception: Inner Exception{Environment.NewLine}   --- End of inner exception stack trace ---", _result["Exception"]);
```

File: src/NLog.StructuredLogging.Json.Tests/LayoutRendererTests.cs [hunk 3]
```diff
@@ -84,10 +84,11 @@ public void WhenConverted_TheResultIsValidJson()
...
+            const string expected = "{\"TimeStamp\":\"2014-01-02T03:04:05.623Z\",\"Level\":\"Error\",\"LoggerName\":\"ExampleLoggerName\",\"Message\":\"{br}This is a message{br}!\\\"£$%^&*{br}{br}With lots of possibly bad things in it\\u0001\\u0002\\u0003\\u0004\\u0005\\u0006\\u0007\\b\\t\\u0010\\u0011\\u0012\\u0013\\u0014\\u0015\\u0016\\u0017\\u0018\\u0019\",\"Exception\":\"System.Exception: Outer Exception ---> System.Exception: Inner Exception{br}   --- End of inner exception stack trace ---\",\"ExceptionType\":\"Exception\",\"ExceptionMessage\":\"Outer Exception\",\"ExceptionStackTrace\":null,\"ExceptionFingerprint\":\"55179621c796d669d13aee15725c01ba4524b44f\",\"Parameters\":\"One,1234\",\"PropertyOne\":\"one\",\"PropertyTwo\":\"2\"}";
```
Comment (martincostello) [inline src/NLog.StructuredLogging.Json.Tests/LayoutRendererTests.cs]: As above.

File: src/NLog.StructuredLogging.Json.Tests/Helpers/MapperStandardTests.cs (hunk 2 above)
Comment (AnthonySteele) [reply src/NLog.StructuredLogging.Json.Tests/Helpers/MapperStandardTests.cs]: Good idea on both of those.

**Comment by martincostello** [comment]
👍

**Pull request merged**